textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/source/IndexSearchSourceOperator.java

```diff
@@ -0,0 +1,28 @@
+package edu.uci.ics.textdb.dataflow.source;

```

Comment:
IIndexSeachSourceOperator interface was already there in the master. Changed it to a class now. So it became part of this pull request.